lib/dep-graph-builders/npm-lock-v2/index.ts

```diff
@@ -18,9 +18,9 @@ import { OutOfSyncError } from '../../errors';
 import { LockfileType } from '../../parsers';
 
 import * as semver from 'semver';
-import * as micromatch from 'micromatch';
 import * as pathUtil from 'path';
 import { eventLoopSpinner } from 'event-loop-spinner';
+import * as multimatch from 'multimatch';
```

Comment:
I changed this import to pass linting.
Is there a reason you used require instead of import ... from?

Reply:
It failed for me locally 🤷🏻